docs/tutorials/tabular_prediction/tabular-indepth.md

```diff
@@ -1,7 +1,7 @@
 # Predicting Columns in a Table - In Depth
 :label:`sec_tabularadvanced`
 
-**Tip**: If you are new to AutoGluon, review :ref:`sec_tabularquick` to learn the basics of the AutoGluon API.
+**Tip**: If you are new to AutoGluon, review :ref:`sec_tabularquick` to learn the basics of the AutoGluon API. To learn how to implement custom models, review :ref:`sec_tabularcustommodel`.
```

Comment:
"implement custom models" may be clearer as:
"add your own custom models to the set that AutoGluon trains, tunes, and ensembles"

Reply:
Maybe also add a FAQ question with this link as the answer such as:

"Does AutoGluon support my favorite model XYZ"?

Reply:
updated and added to FAQ